app/controllers/challenges_controller.rb

```diff
@@ -166,7 +166,7 @@ def update
   end
 
   def destroy
-    return unless challenge.can_delete?
+    return unless @challenge.can_delete?
```

Comment:
I thought "come on, this is just a tiny quick fix for banana, there's no way I could fuck up such a small change, right?" No! I can and did. I am guilty of **hubris**! I must test *every* change. Fuck.

Reply:
Or even better, I need to figure out the issues I've been having with the tests and *write some*. This is the *second* easily-catchable typo bug I've made.